examples/grpo_llama3_demo1.py

```diff
@@ -0,0 +1,1315 @@
+# <a href="https://colab.research.google.com/github/google/tunix/blob/main/examples/grpo_demo.ipynb" ><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>
```

Comment:
you cannot run grpo_llama3_demo1.py or maxtext_model_with_vllm notebook in Tunix w/o installing MaxText, let's not merge those 2 files in Tunix for now